### PR TITLE
BACKLOG-16231: Add an activator to explicit flush sitemap cache on activator

### PR DESCRIPTION
**Pull request by XiangYuHe**
## JIRA

https://jira.jahia.org/browse/BACKLOG-16231

## Description

On activator or deployment on the module it will explicitly refresh the sitemap cache.

### CODE REVIEW

**Review by tdraier — CHANGES_REQUESTED** [review]
It would be more clear and efficient to add the activate method to an existing service (SitemapServiceImpl ?) instead of creating an empty component just for that.

File: src/main/java/org/jahia/modules/sitemap/activators/SitemapActivator.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ *
+ *     Jahia's Enterprise Distributions must be used in accordance with the terms
+ *     contained in the Jahia Solutions Group Terms &amp; Conditions as well as
+ *     the Jahia Sustainable Enterprise License (JSEL).
+ *
+ *     For questions regarding licensing, support, production usage...
+ *     please contact our team at sales@jahia.com or go to http://www.jahia.com/license.
+ *
+ * ==========================================================================================
+ */
```
Comment (rknj) [inline src/main/java/org/jahia/modules/sitemap/activators/SitemapActivator.java:23]: I guess this can removed as the module license is Apache 2

**Review by rknj — CHANGES_REQUESTED** [review]

File: src/main/java/org/jahia/modules/sitemap/activators/SitemapActivator.java (hunk 1 above)
Comment (XiangYuHe) [reply src/main/java/org/jahia/modules/sitemap/activators/SitemapActivator.java:23]: Sure will remove

**Comment by XiangYuHe** [comment]
> It would be more clear and efficient to add the activate method to an existing service (SitemapServiceImpl ?) instead of creating an empty component just for that.

Isn't it more clear if it is not in an existing service?
As the user would know where to find the activator by name

**Review by gflores-jahia — DISMISSED** [review]

**Review by tdraier — APPROVED** [review]

**Review by rknj — APPROVED** [review]

**Pull request merged**